### ISSUE TITLE
Remembering calculations?

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#4037]
I just re-opened an Instat object.  I found (to my pleasure) that the filters are all still there.
But the calculations are not.  
When I have made a few calculations they are remembered for the session.  But I get the impression that they are just as a list and not associated with a particular data set?

Should they be like this, or should they become part of the Instat Object?

**Comment by dannyparsons** [comment]
You mean in the calculator? At the moment they are only in the dialog, and so if you reset, possibly they clear as well? I think the list is associated with the data frame so possibly different lists for different data frames but its not something we store in the instat object at the moment, I thought it was more a dialog thing.

**Comment by rdstern** [comment]
Yes, I was questioning whether calculated columns should have a record of the calculation as part of the meta-data - as is done by Genstat.  So it would not only be remembered in the dialogue.  Also (like filters) it would know which calculation was with which data frame.

![image](https://user-images.githubusercontent.com/11226469/31378093-16b53cc8-ada2-11e7-9961-b617f9415f1a.png)
